db/src/main/resources/db/migration/V1_007__add_address.sql

```diff
@@ -0,0 +1,10 @@
+create table address (
+    id uuid not null primary key,
+    address_line_1 varchar(500) not null,
+    address_line_2 varchar(500),
+    town_or_city varchar(500) not null,
+    postcode varchar(7) not null
+);
+
+alter table claimant add column address_id uuid;
```

Comment:
might want to be specific about what address this is = e.g. card_delivery_address_id